frontend/src/app/map/map.component.html

```diff
@@ -0,0 +1,10 @@
+<div id="map" class="map-container"></div>
+
+<div id="popup" class="ol-popup">
+  <div id="popup-content"></div>
+</div>
+
+<div class="address-container">
+  <input type="text" class="address-input" [(ngModel)]="address" placeholder="{{ 'map.enter' | translate }}">
+  <button class="jump-button" (click)="jumpToAddress()">{{ 'map.jump' | translate }}</button>
```

Comment:
Maybe also use material icons? Otherwise we are using different designs of buttons?